server/startup/migrations/v177.js

```diff
@@ -0,0 +1,14 @@
+import {
+	Migrations,
+} from '../../../app/migrations';
+import {
+	Settings,
+} from '../../../app/models';
+
+
+Migrations.add({
+	version: 177,
+	up() {
+		Settings.updateValueById('Accounts_Default_User_Preferences_sidebarViewMode', 'extended');
```

Comment:
This is not how it is to be done, as we discussed, we'll not change the preference, but rather not use preference and just show extended view when on mobile devices. The preferences can work independently.

Reply:
Some user's prefers medium or condensed. So it should be upto their preference?

Reply:
https://github.com/WideChat/Rocket.Chat/issues/156 This issue was earlier reported by Jay. Which is regarding adding view mode option in Mobile. So we need to finalize it.

Reply:
@Shailesh351 I think we have decided to do it differently for now.  Sorry about the confusion.  Karan is suggesting that it defaults to extended on mobile only.  And there is not option to change it in mobile, no preferences.  But in the web client we would like to leave it as it is upstream.  Thanks.